### ISSUE TITLE
total price not updating 

### ISSUE THREAD

**Issue by SOURABHs23** [cn10xdev/ECommerce_Product_Page#27]
its fixed in my pull request

**Comment by Ezio-Sarthak** [comment]
Hi @SOURABHs23 thanks for bringing up this issue.

Whenever we create an issue we provide a reasonable description for that issue, so that the contributor is able to understand it and work on it. Even if you yourself are working on it, it'd be helpful for others to understand what the issue is, so that no duplicate issues are formed.

Requesting to update the issue title and description. I'm assigning this issue to you

As per the PR for this issue I'm reviewing it now

**Comment by SOURABHs23** [comment]
sorry sir, i thought if fixed any bug then we have to create there on issue tab, i think mistakenly i added so much issues there, please review my PR and i have attached the vedio of what i did

**Comment by Ezio-Sarthak** [comment]
Yeah that's totally fine. Creating issues is not a problem. Not providing a good description for them is. You can take a look at already created issues by me (e.g., this one #1) to understand how to provide context.
